software/contrib/egressus_melodiam.md

```diff
@@ -0,0 +1,101 @@
+# Egressus Melodiam (Stepped Melody)
+
+author: Nik Ansell (github.com/gamecat69)
+date: 02-Feb-24
+labels: clocked lfo, sequencer, CV, randomness
+
+Clockable and free-running LFO and random CV pattern generator.
+Six different wave shapes either define the LFO shape or the slew shape between CV pattern steps.
+Use it to generate variable length melodies (using a quantizer), LFOs or wierd and wonderful control voltages.   
+
+# Inputs, Outputs and Controls
+
+digital_in: clock in
+analog_in: not used
+
+knob_1: Set global CV pattern length / free-running clock speed (A pattern length of 1 produces an cycling LFO)
+knob_2: Set LFO / CV Pattern clock division per output
+
+button_1:
+- Short Press (< 300ms): Cycle through slew/LFO shapes for the selected output
+- Long Press (Between 2 and 5 seconds): Generate a new random CV pattern for the selected output
+
+button_2:
+- Short Press (< 300ms): Cycle through outputs to edit
+- Long Press (Between 2 and 5 seconds): Toggle between clocked mode and free-running mode
+
+output_1: LFO / randomly generated CV pattern (0 to +5V)
+output_2: LFO / randomly generated CV pattern (0 to +5V)
+output_3: LFO / randomly generated CV pattern (0 to +5V)
+output_4: LFO / randomly generated CV pattern (0 to +5V)
+output_5: LFO / randomly generated CV pattern (0 to +5V)
+output_6: LFO / randomly generated CV pattern (0 to +5V)

```

Comment:
[required]
These aren't restricted to 5V anymore; please specify that it's 0 to MAX_OUTPUT_VOLTAGE, as defined by `europi_config`

Reply:
good catch, done